docs/deployment/kustomize.md

```diff
@@ -137,6 +139,28 @@ kubectl apply -k https://github.com/kubernetes-sigs/node-feature-discovery/deplo
 
 ```
 
+# Metrics
```

Comment:
I think the metrics could/should be a separate md file so that users of all deployment methods (kustomize, helm and operator) would find it. All the description about prometheus-operator and the list of exposed metrics would be there.

The metrics section here should be just a short description of the `metrics` overlay and how to use it